packages/github/__tests__/github.test.ts

```diff
@@ -0,0 +1,170 @@
+import * as http from 'http'
+import {GitHub} from '../src/github'
+
+describe('@actions/github', () => {
```

Comment:
New tests to exercise GitHub REST client and GraphQL client with and without proxy

Only runs when env var GITHUB_TOKEN is defined, otherwise warns for discoverability